action.yml

```diff
@@ -48,6 +48,9 @@ inputs:
       GitHub token for API requests.
     required: false
     default: ${{ github.token }}
+  public_github_token:
+    description: |-
+      GitHub token for the public github.com, for use with enterprise GitHub instances.
```

Comment:
```suggestion
      GitHub token for the GitHub instance this action is running on.
    required: false
    default: ${{ github.token }}
  public_github_token:
    description: |-
      GitHub token for github.com. Required to be set when running on a private GitHub Enterprise instance, and ignored otherwise.
```